libs/blocks/marquee/marquee.js

```diff
@@ -64,16 +64,15 @@ const decorateImage = (media) => {
   }
 };
 
-export async function dynamicImport(path) {
-  return import(path);
-}
-
 export async function loadMnemonicList(foreground) {
   try {
-    await dynamicImport('../mnemonic-list/mnemonic-list.js')
-      .then((module) => {
-        module.decorateMnemonicList(foreground);
-      });
+    const { miloLibs, codeRoot } = getConfig();
+    const stylePromise = new Promise((resolve) => {
+      loadStyle(`${miloLibs || codeRoot}/blocks/mnemonic-list/mnemonic-list.css`, resolve);
```

Comment:
You can use `base` here.

Reply:
It is trivial probably but i think since `base` is already there, I think there is no need to initialize an URL instance by parsing an url, thanks for the hint.